### PR TITLE
Add bio for Jame in team page

### PR DESCRIPTION
**Pull request by hongil0316**

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -207,6 +207,15 @@
     and enjoyable for all. When not writing code, Ian enjoys adventures outdoors,
     reading books, and playing his guitar.
 
+- name: James Kwon
+  title: Senior Software Engineer
+  image: /assets/img/headshots/james-kwon.jpg
+  bio: |
+    James is a Software Engineer in Toronto, Ontario. Before working at Gruntwork, 
+    he was a backend engineer in various big tech companies (e.g., Google, Amazaon, Uber).
```
Comment (zackproser) [inline _data/team.yml:215]:
```suggestion
    he was a backend engineer in various big tech companies (e.g., Google, Amazon, Uber).
```

**Comment by ebeneliason** [comment]
It's probably better to re-crop this image as suggested above, but perhaps we should also add `object-fit: cover` to the style for these bio `img` tags so that they clip by default instead of stretching. Here's what that would yield without any change to the photo:
image

**Comment by josh-padnick** [comment]
@hongil0316 Could you carry this across the finish line?

**Comment by hongil0316** [comment]
PTAL. Addressed the comments.

**Comment by hongil0316** [comment]
@josh-padnick @zackproser @ebeneliason PTAL.

**Review by zackproser — APPROVED** [review]
Nice, LGTM!

**Comment by zackproser** [comment]
https://github.com/gruntwork-io/gruntwork-io.github.io/releases/tag/v0.0.198

**Pull request merged**